nvflare/private/fed/server/job_cmds.py

```diff
@@ -19,11 +19,14 @@
 from nvflare.apis.job_def import JobMetaKey
 from nvflare.apis.job_def_manager_spec import JobDefManagerSpec
 from nvflare.fuel.hci.conn import Connection
-from nvflare.fuel.hci.reg import CommandModule, CommandModuleSpec, CommandSpec
+from nvflare.fuel.hci.reg import CommandModuleSpec, CommandSpec
 from nvflare.private.fed.server.server_engine import ServerEngine
+from nvflare.private.fed.server.server_engine_internal_spec import ServerEngineInternalSpec
+from nvflare.security.security import Action
+from .training_cmds import TrainingCommandModule
 
 
-class JobCommandModule(CommandModule):
+class JobCommandModule(TrainingCommandModule):
```

Comment:
why do we want to inherit from TrainingCommandModule?

Reply:
To inherit the auth methods.